### PR TITLE
Updated Screenshots In Docs (close #8883)

### LINKED ISSUE #8883: docs: update screenshots
### Problem description

Screenshots have mostly been updated for Cloud related docs (identified by the ☁️ icon on docs) and for the Enterprise docs (identified by the 🏢 icon on docs). 

A lot of the screenshots of the feature docs are outdated.

### Describe the solution you'd like

Update the updated screenshots.

### Instructions

Please read carefully through the [Wiki on writing documentation](https://hasura.io/docs/wiki/), especially the [images section](https://hasura.io/docs/wiki/style/images/).

### PR DESCRIPTION
**Pull request by Sujithkumardola**
### Description

Updated Screenshots In Databases Section of Docs.

Partially closes #8883 

#### Short Changelog

Screenshots updated to 2.10.1 in Databases section of docs.

#### Long Changelog

 updated Screenshots in
 
 databases/connect-db/index
 databases/postgres/citus-hyperscale-postgres/getting-started/cloud
 databases/postgres/citus-hyperscale-postgres/getting-started/docker
 databases/ms-sql-server/getting-started/cloud
 databases/ms-sql-server/getting-started/docker
 databases/bigquery/getting-started/docker

### Related Issues

Related Issue : #8883

### CODE REVIEW

**Comment by Sujithkumardola** [comment]
Hey @seanparkross I updated the screenshots and you can have a preview here at [https://sujith37.vercel.app](https://sujith37.vercel.app])

If there is any other issue, please let me know.

File: docs/docs/databases/bigquery/getting-started/docker.mdx [hunk 1]
```diff
@@ -87,6 +87,9 @@ fill in the necessary details:
 <Thumbnail src='/img/databases/bigquery/bigquery-add-service-account_console_2.12.1.png' alt='Connect
 existing BigQuery database in Hasura Cloud' width='900px' />
 
+<Thumbnail src='/img/databases/bigquery/connect-big-query-db_console_2.10.1.png' alt='Connect
```
Comment (seanparkross) [inline docs/docs/databases/bigquery/getting-started/docker.mdx:90]: For this image, there is a horizontal scrollbar left in the crop of the page. Please crop this out. Also, if possible, the name of the environment variable here is different to that which is displayed in the image just before it. Which isn't ideal. eg:

Screenshot 2022-10-18 at 14 19 08

File: docs/docs/databases/connect-db/index.mdx [hunk 2]
```diff
@@ -31,7 +31,7 @@ engine instance via the console / metadata APIs / CLI:
 
 Head to `Data -> Manage -> Connect database`
 
-<Thumbnail src='/img/databases/connect-default-db.png' alt='Connect default database' width='1000px' />
+<Thumbnail src='/img/databases/connect-db_step-1_console_2.10.1.png' alt='Connect default database' width='1000px' />
```
Comment (seanparkross) [inline docs/docs/databases/connect-db/index.mdx:34]: Please leave the setting on this image on "Environment Variable" as that is the preferred way of connecting to DBs. 

Please crop like the existing image as per the original and not the whole page. EG:
Screenshot 2022-10-18 at 14 21 55

File: docs/docs/databases/ms-sql-server/getting-started/cloud.mdx [hunk 3]
```diff
@@ -59,17 +59,17 @@ Once you add the database, you'll see your database pop up on the sidebar.
...
 list of tables.
 
-<Thumbnail src='/img/databases/ms-sql-server/3-manage-mydb.png' alt='Manage my-db' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/manage-db_step-3_console_2.10.1.png' alt='Manage my-db' width='1000px' />
 
 Track tables selectively or all of them so that Hasura can introspect the tables and create the corresponding GraphQL
 schema.
 
-<Thumbnail src='/img/databases/ms-sql-server/4-track-tables.png' alt='Track tables' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/track-tables_step-3_console_2.10.1.png' alt='Track tables' width='1000px' />
```
Comment (seanparkross) [inline docs/docs/databases/ms-sql-server/getting-started/cloud.mdx:67]: This is too wide. Please set the width so it's the same size as the actual UI. 


Screenshot 2022-10-18 at 14 26 29

File: docs/docs/databases/ms-sql-server/getting-started/cloud.mdx [hunk 4]
```diff
@@ -59,17 +59,17 @@ Once you add the database, you'll see your database pop up on the sidebar.
...
 
 If you have foreign keys, you'll also see suggested relationships. Again, you can choose to track them selectively or
 all at once.
 
-<Thumbnail src='/img/databases/ms-sql-server/5-track-rels.png' alt='Track relationships' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/track-foreign-key-rel_step-3_console_2.10.1.png' alt='Track relationships' width='1000px' />
```
Comment (seanparkross) [inline docs/docs/databases/ms-sql-server/getting-started/cloud.mdx:72]: Also too wide. Set to actual size of the UI. Any chance we could rename "People" to "Users" it looks more professional. 

Screenshot 2022-10-18 at 14 28 41

File: docs/docs/databases/ms-sql-server/getting-started/docker.mdx [hunk 5]
```diff
@@ -96,7 +96,7 @@ container on Linux, Mac or Windows.
 
 <Thumbnail src='/img/databases/ms-sql-server/1-manage-databases.png' alt='Manage databases' width='1000px' />
```
Comment (seanparkross) [inline docs/docs/databases/ms-sql-server/getting-started/docker.mdx:97]: Can you also do this image? It's now out of place with reference to the others.

File: docs/docs/databases/ms-sql-server/getting-started/docker.mdx [hunk 6]
```diff
@@ -105,17 +105,17 @@ Once you add the database, you'll see your database pop up on the sidebar.
...
 list of tables.
 
-<Thumbnail src='/img/databases/ms-sql-server/3-manage-mydb.png' alt='Manage my-db' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/manage-db_step-3_console_2.10.1.png' alt='Manage my-db' width='1000px' />
 
 Track tables selectively or all of them so that Hasura can introspect the tables and create the corresponding GraphQL
 schema.
 
-<Thumbnail src='/img/databases/ms-sql-server/4-track-tables.png' alt='Track tables' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/track-tables_step-3_console_2.10.1.png' alt='Track tables' width='1000px' />
```
Comment (seanparkross) [inline docs/docs/databases/ms-sql-server/getting-started/docker.mdx:113]: Too wide.

File: docs/docs/databases/ms-sql-server/getting-started/docker.mdx [hunk 7]
```diff
@@ -105,17 +105,17 @@ Once you add the database, you'll see your database pop up on the sidebar.
...
 
 If you have foreign keys, you'll also see suggested relationships. Again, you can choose to track them selectively or
 all at once.
 
-<Thumbnail src='/img/databases/ms-sql-server/5-track-rels.png' alt='Track relationships' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/track-foreign-key-rel_step-3_console_2.10.1.png' alt='Track relationships' width='1000px' />
```
Comment (seanparkross) [inline docs/docs/databases/ms-sql-server/getting-started/docker.mdx:118]: Too wide. Also could "People" be renamed "Users"

**Review by seanparkross — CHANGES_REQUESTED** [review]

**Comment by Sujithkumardola** [comment]
Hey @seanparkross, I made the changes

**Comment by Sujithkumardola** [comment]
> Hey @seanparkross, I made the changes

have a preview here at [https://sujith37.vercel.app](https://sujith37.vercel.app]/)

**Review by seanparkross — APPROVED** [review]
Thanks! Looks good

**Comment by seanparkross** [comment]
@Sujithkumardola sorry I hate to be the one to say this but there seems to be a red heart lock on the form fields in some of your shots. I'm not sure if I missed this last time. Sorry if I did. I'm not sure what it is. Maybe a browser extension?


connect-db-cloud-cb5e0b81ade17fcf01a141323732208c

**Comment by Sujithkumardola** [comment]
> @Sujithkumardola sorry I hate to be the one to say this but there seems to be a red heart lock on the form fields in some of your shots. I'm not sure if I missed this last time. Sorry if I did. I'm not sure what it is. Maybe a browser extension?
> 
> 
> connect-db-cloud-cb5e0b81ade17fcf01a141323732208c
> 
> 

As mentioned in the linked issue, I only updated the feature documentation and not the Cloud documentation.

**Comment by Sujithkumardola** [comment]
> @Sujithkumardola sorry I hate to be the one to say this but there seems to be a red heart lock on the form fields in some of your shots. I'm not sure if I missed this last time. Sorry if I did. I'm not sure what it is. Maybe a browser extension?
> 
> 
> connect-db-cloud-cb5e0b81ade17fcf01a141323732208c
> 
> 

Could you please specify the file location?

**Comment by Stefmore02** [comment]
Hey! @Sujithkumardola 

Thanks so much for your contribution to Hasura OSS during the Hacktoberfest 2022! We'd love to send you some Hasura swag as a token of appreciation and you will be getting an email with the swag form shortly.✨

**Pull request closed without merging**